matrix_client/client.py

```diff
@@ -127,6 +127,10 @@ def start_listener_thread(self, timeout=30000):
             e = sys.exc_info()[0]
             print("Error: unable to start thread. " + str(e))
 
+    def upload(self,content,content_type):
+        response = self.api.media_upload(content,content_type)
+        return response["content_uri"]
```

Comment:
Gotcha, I didn't think about the missing key use case.
I know it will throw an error if a 2XX code isn't returned, but even that isn't specific.